pallets/subtensor/src/weights.rs

```diff
@@ -115,10 +123,14 @@ impl<T: Config> Pallet<T> {
                 version_key,
             ));
```

Comment:
should we add nonce into hash computation.

Reply:
done